moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -827,6 +827,30 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
     }
   }
 
+  MoveItErrorCode plan(Plan& plan_, const std::string& pipeline_id, const std::string& planner_id)
```

Comment:
Since `plan` is a callable below, maybe use a more different variable name here instead of just appending an underscore? Like `output_plan`?